src/docs/asciidoc/getting_started/index.adoc

```diff
@@ -94,10 +94,8 @@ section of the reference documentation.
 
 ----
 
-NOTE: If you open the json file of the card, you will see '${current_date_in_milliseconds_from_epoch}' for the field 'startDate'.
-      We have used this so that the date of the card is the current day (or the next day in some other examples). Indeed, in the shell script
-      that sends the card, you will see that we create an environment variable with the current date which is then injected
-      into the json file.
+NOTE: If you open the json file of the card, you will see that values of date fields ('startDate', 'endDate', etc..) are not epoch values (number of milliseconds from 01/01/1970) but relative values.
+      We have used this for testing so that the date of the card is the current day (or the next day in some other examples). Indeed, in the shell script that sends the card, relative dates are converted in epoch dates and then injected into the json file.
```

Comment:
We do not use yet the cli on getting started so this document shall not be modified